### DISCUSSION TITLE
Problems when accessing private organization

### DISCUSSION THREAD

**Discussion by LimingFang** [magit/forge#531] [Support] [answered]
- `Magit 20221125.21 [>= 3.3.0-git], Git 2.20.1, Emacs 30.0.50, gnu/linux`
- `ghub 20221201.1411`
- `forge 20221217.2216`
- debian-10
---
**Goal: i want to use forge with my company's gitlab.**   

- I have set `.authinfo` as following: ( the * is my TOKEN)
  > machine code.byted.org/api/v4 login fangliming.xxm^forge password ****

- i have also set the `gitlab.HOST.user` as well as `gitlab.host` variable:
  <img width="567" alt="image" src="https://user-images.githubusercontent.com/28479651/210129513-a6051367-6b25-495e-85a6-94ffbc758423.png">

- i also use Git URL rewriting instead: 
  <img width="421" alt="image" src="https://user-images.githubusercontent.com/28479651/210129544-2d053998-a590-4ec5-a309-10eac4b57eae.png">

- and i don't change the `forge-alist` variable.

---

But when i opened a repo's magit status buffer, i typed `f n`, and i got:
> Cannot determine forge repository.  git@code.byted.org:storage/byte.git isn’t a forge url

**Comment by tarsius** [comment, marked as answer]
> and i don't change the `forge-alist` variable.

I **do** have to add a new entry to that variable.
